src/services/bindService.js

```diff
@@ -74,6 +74,90 @@ angular.module("openshiftCommonServices")
       return binding;
     };
 
+    var isServiceBindable = function(serviceInstance, serviceClasses) {
+      var serviceClass = getServiceClassForInstance(serviceInstance, serviceClasses);
+      if (!serviceClass) {
+        return !!serviceInstance;
+      }
+
+      // If being deleted, it is not bindable
+      if (_.get(serviceInstance, 'metadata.deletionTimestamp')) {
+        return false;
```

Comment:
Edge case here where this will return true if serviceClass is not defined above. We should move this above the service class check.